maven/bnd-resolver-maven-plugin/src/main/java/aQute/bnd/maven/resolver/plugin/ResolverMojo.java

```diff
@@ -76,6 +78,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				resolve(runFile, fileSetRepository);
 			}
 		} catch (Exception e) {
+			if (e instanceof ResolutionException) {
```

Comment:
What about also https://github.com/bndtools/bnd/blob/master/maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java#L84 and https://github.com/bndtools/bnd/blob/master/maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java#L111?